examples/example3/README.md

```diff
@@ -0,0 +1,40 @@
+## Example 3 – Using virtual resources and collection in a master-agent set-up.
+
+
+This is an (almost) complete example for a master-agent set-up using virtual resources in Puppet.
```

Comment:
It might be worth making a note here that services are in this module are not be expected to be exported, only the host here.  Just as a measure to avoid confused users, or poorly performing monitoring nodes.